<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CombineSyncEmbeddedWidget</class>
 <widget class="QWidget" name="CombineSyncEmbeddedWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>2</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="operationLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="operationLabel">
       <property name="text">
        <string>Op:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="mpComboBox">
       <item>
        <property name="text">
         <string>AND</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OR</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="inputSizeLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="inputSizeLabel">
       <property name="text">
        <string>Inputs:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="mpSpinBox">
       <property name="toolTip">
        <string>Number of input ports</string>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="mpResetButton">
     <property name="toolTip">
      <string>Reset all ready states and sync values</string>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
